TLM/TMPE.API/Traffic/Enums/KeyAttribute.cs

```diff
@@ -0,0 +1,22 @@
+namespace TrafficManager.API.Traffic.Enums {
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
+    public class KeyAttribute : Attribute {
```

Comment:
Would this be a bit more descriptive if named `LocaleKeyAttribute`?

Reply:
yeah. maybe we can add a second field for tooltip too if/when the need arises.